cmd/kpod/inspect.go

```diff
@@ -83,7 +83,10 @@ func inspectCmd(c *cli.Context) error {
 	}
 	server, err := libkpod.New(config)
 	if err != nil {
-		return err
+		return errors.Wrapf(err, "could not get container server")
+	}
+	if err = server.Update(); err != nil {
```

Comment:
Don't we call this in libkpod.New()? Very strange that it needs to be called again here.